frontend/src/patientApp/timeOfTest/PatientFormContainer.tsx

```diff
@@ -93,7 +93,7 @@ const PatientFormContainer = () => {
                 <Button
                   id="edit-patient-save-lower"
                   disabled={!formChanged}
-                  onClick={onSave}
+                  onClick={() => onSave()}
```

Comment:
thanks for changing this too! we should probably just remove this who timeOfTest folder, ill make a new ticket for that